### PR TITLE
Login Failure Handling 

### PR DESCRIPTION
**Pull request by mwodahl**
Adds support for 400-range login failure errors in the webapp

### CODE REVIEW

**Review by drewjj — CHANGES_REQUESTED** [review]
Everything is looking good but you need to update the unit tests. They are failing for all 3 of these files that you have modified now. Feel free to reach out if you need guidance.

**Comment by mwodahl** [comment]
Unit tests for the changed files have been updated and are now passing

**Review by drewjj — APPROVED** [review]
Tests all pass and I think the handling of the errors is nice. Thanks!

File: webapp/src/generalSlices/userSlice.test.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ describe('async thunks', () => {
   const initialState = {
     loading: null,
     bsmLoading: null,
+    loginMessage: "",
```
Comment (payneBrandon) [inline webapp/src/generalSlices/userSlice.test.js]: Let's use that prettierrc plugin to format here and get these double quotes to conform to the single quite standard we've selected (ctrl+k + ctrl+d shortcut in vscode to format)

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one comment, looks good!

File: webapp/src/generalSlices/userSlice.test.js (hunk 1 above)
Comment (mwodahl) [reply webapp/src/generalSlices/userSlice.test.js]: Updated to use single quotes

**Review by payneBrandon — APPROVED** [review]

**Pull request merged**